src/widget/wlabel.cpp

```diff
@@ -43,26 +45,62 @@ void WLabel::setup(QDomNode node, const SkinContext& context) {
     setPalette(pal);
 
     // Text
-    if (context.hasNode(node, "Text"))
-        m_qsText = context.selectString(node, "Text");
-    setText(m_qsText);
+    if (context.hasNodeSelectString(node, "Text", &m_skinText)) {
+        setText(m_skinText);
+    }
 
     // Font size
-    if (context.hasNode(node, "FontSize")) {
-        int fontsize = context.selectString(node, "FontSize").toInt();
+    QString strFontSize;
+    if (context.hasNodeSelectString(node, "FontSize", &strFontSize)) {
+        int fontsize = strFontSize.toInt();
         setFont(QFont("Helvetica", fontsize, QFont::Normal));
```

Comment:
TODO: should we change the default font to Open Sans since we bundle it, and only Mac comes with Helvetica?